src/Metaport.tsx

```diff
@@ -22,39 +22,21 @@
  */
 
 // @ts-ignore
-import React from 'react';
-import { createRoot } from 'react-dom/client';
+import React from 'react'
+// import { createRoot } from 'react-dom/client';
+import * as interfaces from './core/interfaces'
 
-import { Widget } from './components/Widget';
-import { internalEvents } from './core/events';
-
-import * as interfaces from './core/interfaces/index';
-export * as dataclasses from './core/dataclasses/index';
-export * as interfaces from './core/interfaces/index';
-
-export * as sfuel from './core/sfuel';
-
-
-export class Metaport {
+export class InjectedMetaport {
   constructor(config: interfaces.MetaportConfig) {
-    if (config.openButton === undefined) config.openButton = true;
-    if (config.autoLookup === undefined) config.autoLookup = true;
-    if (config.skaleNetwork === undefined) config.skaleNetwork = 'mainnet';
-    if (config.debug === undefined) config.debug = false;
-    createRoot(document.getElementById('metaport')).render(<Widget config={config} />);
+    if (config.openButton === undefined) config.openButton = true
+    if (config.autoLookup === undefined) config.autoLookup = true
+    if (config.skaleNetwork === undefined) config.skaleNetwork = 'mainnet'
+    if (config.debug === undefined) config.debug = false
+    const el = document.getElementById('metaport')
+    if (el) {
+      // createRoot(el).render(<Widget config={config} />);
+    } else {
+      console.log('div with id="metaport" does not exist')
+    }
   }
```

Comment:
No element is being created here. Confirming this is correct?
Also would say ideally due to the empty commented if, if it is correct to switch to an if only where it is more like 

```javascript
if (el === undefined) {
      console.log('div with id="metaport" does not exist')
}
```

Reply:
this one will utilised later, once we will have a standalone build of Metaport